migrations/versions/2018_12_05_remove_constraints.py

```diff
@@ -0,0 +1,26 @@
+"""Remove redundant constraints and rename from uix -> uq to follow naming convention
+
+Revision ID: 2018_12_05_remove_constraints
+Revises: 2018_12_05_rename_constraint
+Create Date: 2018-12-05 18:43:24.009236
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "2018_12_05_remove_constraints"
+down_revision = "2018_12_05_rename_constraint"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_unique_constraint("uq_page_guid_version", "page", ["guid", "version"])
```

Comment:
:+1: better to stick to what's possible with Alembic rather than using the Postgres-specific `ALTER TABLE ... RENAME CONSTRAINT`